content/integration/gnome-terminal.md

```diff
@@ -1,12 +1,7 @@
 ---
-title: "Using gnome-terminal with rvm"
+title: "Using gnome-terminal with RVM"
 ---
 
-Other terminals: [LXTerminal](/integration/lxterminal/),
-[XTerm](/integration/xterm/), [Konsole](/integration/konsole/),
-[Zsh + iTerm](/integration/zsh/#zsh-iterm) and
-[anything else](/integration/).
```

Comment:
Those are here as this `gnome-terminal` page is linked in few places, including SO and RVM error messages, this way users can find their solution faster - instead of arguing they do not use gnome-terminal